doc/tasks/deptrac.md

```diff
@@ -0,0 +1,51 @@
+# Deptrac
+
+Follow the [installation instructions](https://github.com/sensiolabs-de/deptrac#installation) to add deptrac to your 
```

Comment:
The project itself [recommends installing the phar file](https://github.com/sensiolabs-de/deptrac#download-the-phar-recommended). 

Also, the package does not come with a bin file. I guess this should also work when you run `php deptrac.php` in the package, but I didn't look further into that option.